libs/blocks/featured-article/featured-article.css

```diff
@@ -0,0 +1,189 @@
+}
+
+@media (min-width: 600px) {
+  .featured-article-card-image {
+    width: 40%;
+  }
+}
+
+@media (min-width: 1200px) {
+  .featured-article-card-image {
+    width: 60%;
+  }
+}
+
+.featured-article-card-image img {
+  object-fit: cover;
+  height: 100%;
+  width: 100%;
+  transition: .4s;
+}
+
+.featured-article-card-image img:hover {
+  opacity: 0.6;
+}
+
+.featured-article-card-body {
+  padding: 2rem;
+  text-align: center;
+}
+
+@media (min-width: 600px) {
+  .featured-article-card-body {
+    width: 60%;
+    text-align: left;
+  }
+}
+
+@media (min-width: 1200px) {
+  .featured-article-card-body {
+    width: 40%;
+  }
+}
+
+.featured-article-card-body p,
+.featured-article-card-body h3 {
+  margin-left: 0;
+  margin-right: 0;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.featured-article-card-category,
+.featured-article-card-category a,
+.featured-article-card-date {
+  font-weight: var(-type-detail-all-weight);
+  color: var(--type-detail-color);
+  font-size: var(--type-detail-s-lh);
+  line-height: var(--type-detail-line-height);
+  text-transform: uppercase;
+  letter-spacing: .1em;
+  margin-bottom: 0.5rem;
+}
+
+.featured-article-card-body h3 {
+  font-size: var(--type-heading-l-size);
+  line-height: var(--type-heading-line-height);
+  margin: 0;
+  margin-bottom: 1rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+@media (min-width: 600px) {
+  .featured-article-card-body h3 {
+    font-size: var(--type-heading-xl-size);
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
+  }
+}
+
+.featured-article-card-body .featured-article-card-description {
+  color: var(--color-gray-700);
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+@media (min-width: 600px) {
+  .featured-article-card-body .featured-article-card-description {
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+  }
+}
+
+/* TAGS PAGE */
+.tag-header-container .featured-article-card {
+  background: var(--color-white);
+  margin: 2rem;
+  padding: 16px;
+  border-radius: 4px;
+  border: solid 1px var(--color-gray-200);
+  text-decoration: none;
+}
+
+@media (min-width: 600px) {
+  .tag-header-container .featured-article-card {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 800px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .tag-header-container .featured-article-card {
+    max-width: calc(1200px - 2rem);
+  }
+}
+
+.tag-header-container .featured-article-card div {
+  text-align: left;
+  padding: 0;
+}
+
+.tag-header-container .featured-article-card-image {
+  max-height: 180px;
+}
+
+@media (min-width: 600px) {
+  .tag-header-container .featured-article-card-image {
+    max-height: unset;
+    margin-right: 16px; 
+  }
+}
+
+.tag-header-container .featured-article-card-category {
+  margin: 2rem 0 16px;
+}
+
+.tag-header-container .featured-article-card-body p:not(:first-child) {
+  font-size: var(--type-body-font-size-xs);
```

Comment:
This font variable is not declared. I wouldn't add a variable here though. Just replace the line with: 
`font-size: 0.875rem;`